packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -1205,13 +1205,13 @@ export const UnifiedDataTable = ({
               {searchDescription ? (
                 <FormattedMessage
                   id="unifiedDataTable.searchGenerationWithDescriptionGrid"
-                  defaultMessage="Table generated by search {searchTitle} ({searchDescription})"
+                  defaultMessage="Table generated by Discover Session ''{searchTitle}'' ({searchDescription})"
```

Comment:
@logeekal I'm unfamiliar with this code, do you know if we use this in Timeline? Show it say `Discover Session` here?

Reply:
thanks!